Cargo.toml

```diff
@@ -29,10 +29,13 @@ lazy_static = "1.4"
 rand = "0.8"
 rand_chacha = "0.3"
 serde = "1"
+serde_bytes = "0.11.15"
 
 [dev-dependencies]
 assert_matches = "1.5.0"
+bitcoin = "0.28.2"
 ed25519-dalek = "2.1"
 ic-vetkd-utils = { git = "https://github.com/dfinity/ic" }
 k256 = "0.13.4"
-pocket-ic = "5.0.0"
+pocket-ic = "6.0.0"
+secp256k1 = { version = "0.22", features = ["global-context", "rand-std"] }
```

Comment:
Yes, I just copy-pasted what Jack (IIRC) used in `ic-crypto-secp256k1`. I bumped both crate versions and also added a test for signature verification with different message lengths. The downside is that `bitcoin` uses a lower version of `secp256k1` and we need to convert one of the `struct`s produced by `bitcoin`, which looks quite ugly. But since this is just a test and this gives us the possibility to test the verification of messages of length !=32, I would say this is bearable.  We could remove the conversion once `secp256k1` is updated in `bitcoin`

Reply:
Thanks for bumping the versions and for testing different message length!

Indeed, testing messages of length !=32 is valuable, so temporarily converting between the different XOnlyPublicKeys seems OK.